ui/packages/catalog-ui-search/src/main/webapp/react-component/container/map-actions/map-actions.tsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import { hot } from 'react-hot-loader'
+import * as React from 'react'
+const wreqr = require('../../../js/wreqr.js')
+const _ = require('underscore')
+
+type Props = {
+  model: Backbone.Model
+}
+
+type State = {
+  currentOverlayUrl: string
+}
+
+const getActionsWithIdPrefix = (actions: any, id: any) => {
+  return actions.filter((action: any) => action.get('id').indexOf(id) === 0)
```

Comment:
Can we use [String.prototype.startsWith()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith)?